### DISCUSSION TITLE
IT天空 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#184] [模板讨论]
IT天空 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by jerrell851** [comment]
从8月21日开始签到失败，错误：

Failed at 2/4 request,
Fail assert: {"re": "errorcode", "from": "content"} from failed_asserts,
Request URL: https://api.itsk.com/webapi/sign

**Reply by wjf0214** [reply]
我这没问题，更新下cookie看看。

**Comment by infisz** [comment]
2023-09-04 18:52:21
日志：Failed at 2/4 request,
Fail assert: {"re": "errorcode", "from": "content"} from failed_asserts,
Response Error : HTTP 400: Bad Request,
Request URL: https://api.itsk.com/webapi/sign

**Comment by goxofy** [comment]
> 2023-09-04 18:52:21 日志：Failed at 2/4 request, Fail assert: {"re": "errorcode", "from": "content"} from failed_asserts, Response Error : HTTP 400: Bad Request, Request URL: https://api.itsk.com/webapi/sign

同样的问题

**Reply by goxofy** [reply]
@wjf0214 大佬这个有办法吗

**Comment by xuanxuaner** [comment]
2023-11-25 01:19:13
日志：Failed at 2/4 request,
Fail assert: {"re": "errorcode", "from": "content"} from failed_asserts,
Response Error : HTTP 400: Bad Request,
Request URL: https://api.itsk.com/webapi/sign

**Reply by wjf0214** [reply]
网站签到有验证，签到框架不好生成处理。我另起了一个 [脚本库](https://github.com/wjf0214/ql-script)，刚加了IT天空，是用青龙跑的，可以试试。
不过python不是很熟，算是边学边写的。有问题也可以反馈。
